cmd/gen-manifests/main.go

```diff
@@ -121,7 +121,14 @@ func makeManifestJob(name string, imgType distro.ImageType, cr composeRequest, d
 		}
 	}
 	job := func(msgq chan string) error {
```

Comment:
You can switch to

```
func(msgq chan string) (err error)
```

and then drop `success` and just rely on `err != nil`. No hard feelings about this though.